rllib/agents/dqn/dqn_torch_model.py

```diff
@@ -87,9 +87,9 @@ def __init__(
             advantage_module.add_module("A", nn.Linear(ins, action_space.n))
             value_module.add_module("V", nn.Linear(ins, 1))
         # Non-dueling:
-        # Q-value layer (use Advantage module's outputs as Q-values).
+        # Q-value layer (use main module's outputs as Q-values).
```

Comment:
This was actually a torch bug in DQN, where the output was once again pushed through yet another Linear. We do not do this in tf and should not do this in torch either, of course.